tests/test_standard_input_json.py

```diff
@@ -1,23 +1,169 @@
+import datetime
+import json
 import os
+import tempfile
 import unittest
+import uuid
+import logging
 
+from contextlib import contextmanager
 from pathlib import Path
 
+import hca
+import jsonschema
+import requests
+
+from loader.schemas import standard_schema
+from loader.standard_loader import SCHEMA_URL
+from scripts.cgp_data_loader import main as cgp_data_loader_main
+from tests import eventually, ignore_resource_warnings, message
+
+logger = logging.getLogger(__name__)
+
+TEST_DATA_PATH = Path(__file__).parents[1] / 'tests' / 'test_data'
+
 
 class TestStandardInputFormatLoading(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.dss_client = hca.dss.DSSClient()
+        cls.dss_client.host = 'https://hca-dss-4.ucsc-cgp-dev.org/v1'
         cls.dss_endpoint = os.getenv("TEST_DSS_ENDPOINT", "https://hca-dss-4.ucsc-cgp-dev.org/v1")
         cls.staging_bucket = os.getenv("DSS_S3_STAGING_BUCKET", "mbaumann-dss-staging")
-        cls.project_path = Path(__file__).parents[1]
+        cls.test_file = TEST_DATA_PATH / 'gen3_sample_input_standard_metadata.json'
+
+    def test_data_matches_schema(self):
```

Comment:
Ideally there should also be test(s) for the case in which the input data file does not contain all of the required fields. What happens when one (or more) bundles in the input file do not contain all of the required fields?